src/models/remote_content.ts

```diff
@@ -0,0 +1,112 @@
+import * as t from "io-ts";
+
+import {
+  FiscalCode,
+  NonEmptyString,
+  Ulid
+} from "@pagopa/ts-commons/lib/strings";
+import { enumType } from "@pagopa/ts-commons/lib/types";
+import { Container } from "@azure/cosmos";
+import {
+  CosmosdbModelVersioned,
+  RetrievedVersionedModel
+} from "../utils/cosmosdb_model_versioned";
+import { Has_preconditionEnum } from "../../generated/definitions/ThirdPartyData";
+import { ServiceId } from "../../generated/definitions/ServiceId";
+
+export const RC_CONFIGURATION_COLLECTION_NAME = "remote-content-configuration";
+
+const RC_CONFIGURATION_MODEL_PK_FIELD = "configurationId";
+
+export const RCClientCert = t.interface({
+  clientCert: NonEmptyString,
+  clientKey: NonEmptyString,
+  serverCa: NonEmptyString
+});
+
+const RCAuthenticationDetails = t.interface({
+  headerKeyName: NonEmptyString,
+  key: NonEmptyString,
+  type: NonEmptyString
+});
+
+export type RCAuthenticationConfig = t.TypeOf<typeof RCAuthenticationConfig>;
+export const RCAuthenticationConfig = t.intersection([
+  RCAuthenticationDetails,
+  t.partial({ cert: RCClientCert })
+]);
+
+export type RCEnvironmentConfig = t.TypeOf<typeof RCEnvironmentConfig>;
+export const RCEnvironmentConfig = t.interface({
+  baseUrl: NonEmptyString,
+  detailsAuthentication: RCAuthenticationConfig
+});
+
+export type RCTestEnvironmentConfig = t.TypeOf<typeof RCTestEnvironmentConfig>;
+export const RCTestEnvironmentConfig = t.intersection([
+  t.interface({
+    testUsers: t.readonlyArray(FiscalCode)
+  }),
+  RCEnvironmentConfig
+]);
+
+const RCConfigurationR = t.interface({
+  configurationId: Ulid,
+  description: NonEmptyString,
+  disableLollipopFor: t.readonlyArray(FiscalCode),
+  hasPrecondition: enumType<Has_preconditionEnum>(
+    Has_preconditionEnum,
+    "hasPrecondition"
+  ),
+  isLollipopEnabled: t.boolean,
+  name: NonEmptyString,
+  serviceId: ServiceId,
```

Comment:
ServiceId has to be removed

Reply:
[abbc084](https://github.com/pagopa/io-functions-commons/pull/357/commits/abbc084ff82eb5791ecbf3ecfe67407b10a46160)